.gitlab/Dockerfile-2.7.8

```diff
@@ -0,0 +1,91 @@
+# This is copied from official Docker image, but compile Ruby with `--disable-shared` instead.
```

Comment:
May be worth leaving a link here to the upstream file, so we can pick up changes in the future?